addon/components/frost-component.js

```diff
@@ -31,19 +31,23 @@ export default Component.extend(PropTypesMixin, HookMixin, SpreadMixin, CssMixin
       PropTypes.null
     ]),
     attributeBindings: PropTypes.arrayOf(PropTypes.string),
-    classNameBindings: PropTypes.arrayOf(PropTypes.string),
-    classNames: PropTypes.arrayOf(PropTypes.string),
+    // FIXME: the following property defined in propTypes cause validation errors
+    // when ember-prop-types validateOnUpdate setting is set to true
+    // classNameBindings: PropTypes.arrayOf(PropTypes.string),
+    // classNames: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I think https://github.com/ciena-blueplanet/ember-prop-types/issues/99 should end up fixing this.